config/initializers/session_store.rb

```diff
@@ -21,7 +21,7 @@
                 when "memory" then  :memory_store
                 when "cache" then   :dalli_store
                 else
-                  raise "session_store, '#{evm_store}', invalid. Should be one of 'sql', 'memory', 'cache'"
+                  raise "session_store, '#{evm_store}', invalid. Should be one of 'sql', 'memory', 'cache'.  Source configuration: #{config_file}"
```

Comment:
Yes, if we write another invalid `vmdb.yml.db` somewhere else in test or code, I want the error to lead someone to say, "hey, I have a vmdb.yml.db with nearly nothing in it"